--- os-bases/centos/usr/lib/contractor/resources/centos.toml	2019-07-15 07:15:37.359929201 -0600
+++ os-bases/centos/usr/lib/contractor/resources/centos0.toml	2019-07-15 07:15:26.747910189 -0600
@@ -13,7 +13,7 @@
 [ structure.centos-6-base.config_values ]
   distro_version = '6'
   installer_pxe = 'centos-6'
-  vcenter_guest_id = 'centos6_64Guest'
+  vcenter_guest_id = 'rhel6_64Guest'
   azure_image = { 'publisher' = 'OpenLogic', 'offer' = 'CentOS', 'sku' = '6.10', 'version' = 'latest' }
 
 [ structure.centos-7-base ]
@@ -23,7 +23,7 @@
   memory_size = 2048  # centos 7 requires more minimum memory to install
   distro_version = '7'
   installer_pxe = 'centos-7'
-  vcenter_guest_id = 'centos7_64Guest'
+  vcenter_guest_id = 'rhel7_64Guest'
   azure_image = { 'publisher' = 'OpenLogic', 'offer' = 'CentOS', 'sku' = '7.6', 'version' = 'latest' }
 
 [ pxe.centos-6 ]

